pyhomematic/devicetypes/actors.py

```diff
@@ -183,13 +183,22 @@ class IPKeyDimmer(GenericDimmer, HelperActionPress):
     def __init__(self, device_description, proxy, resolveparamsets=False):
         super().__init__(device_description, proxy, resolveparamsets)
 
+        channels = []
+        if "HMIP-DRDI3" in self.TYPE.upper():
+            channels = [1,2,3]
+        elif "HMIP-BDT" in self.TYPE.upper():
```

Comment:
Please just use `else` here so we have channels `1` and `2` as the defaults for all other key-dimmers. Also please use whitespace like everywhere else in the code (`[1, 2, 3]`, not `[1,2,3]`) so the code-style is consistent across the project.